org-roam.el

```diff
@@ -231,15 +231,52 @@ If FILE is not specified, use the current buffer's file-path."
          (f-descendant-of-p (file-truename path)
                             (file-truename org-roam-directory))))))
 
-(defun org-roam--list-files (dir)
-  "Return all Org-roam files located within DIR, at any nesting level.
-Ignores hidden files and directories."
+(defun org-roam--shell-command-files (cmd)
+  "Run CMD in the shell and return a list of files. If no files are found, an empty list is returned."
+  (seq-filter #'s-present? (split-string (shell-command-to-string cmd) "\n")))
```

Comment:
Probably want to `shell-quote-argument` before executing `cmd`

Reply:
I think it's not relevant to this use case, since I'm sending characters that I need the shell to actually treat as special characters (`"*.org"`) and not be escaped. When I use `shell-quote-argument`, the function doesn't work as expected.